docker-compose.yaml

```diff
@@ -15,6 +15,7 @@ services:
       VECTOR_EMBEDDER_BASIC_AUTH_PASSWORD: $VECTOR_EMBEDDER_BASIC_AUTH_PASSWORD
       VECTOR_STORE_BASIC_AUTH_PASSWORD: $VECTOR_STORE_BASIC_AUTH_PASSWORD
       GF_FEATURE_TOGGLES_ENABLE: externalServiceAccounts
+      NOT_HG: true
```

Comment:
Maybe DEV_MODE or something instead? The double negative can be a bit confusing, and more accurately represents the state.